games/g.yaml

```
@@ -259,6 +259,27 @@
   added: 2025-08-26
   updated: 2025-08-26
 
+- name: Glider for Apple II
+  originals:
+  - Glider
+  type: remake
+  repo: 'https://github.com/colinleroy/a2tools/tree/master/src/glider'
```

```suggestion
  repo: https://github.com/colinleroy/a2tools
```
osgc doesn't handle monorepos well; instead we can have a single project associated with multiple original games